docs/source/advancedcommands.rst

```diff
@@ -1,34 +1,12 @@
 Advanced Commands
 *****************
 
-Feasibility Analysis
+Relaxation Analysis
 ====================
 
 If your Model doesn't solve, you can automatically find the nearest feasible version of it with the ``Model.feasibility()`` command, as shown below. The feasible version can either involve relaxing all constraints by the smallest number possible (that is, dividing the less-than side of every constraint by the same number), relaxing each constraint by its own number and minimizing the product of those numbers, or changing each constant by the smallest total percentage possible.
```

Comment:
text above still references `Model.feasibility()` and may be out of date.

Also, a sidenote: for real applications, I think we should promote putting the top-level specs in a named constraint set (e.g. "Specifications" or "Requirements"). This way, it will be super easy to only relax the top-level requirements (range, payload, takeoff length, etc) while not relaxing physical relationships that aren't appropriate to relax. This could make a nice example! It also has the benefit of removing the "choice" of what variable should be the objective; instead, set targets for each of the requirements/specifications, and always solve a Feasibility formulation (which will do better than the targets if they were feasible to begin with).

Sorry for the long comment, just brain dumping